commands.js

```diff
@@ -0,0 +1,29 @@
+exports.list = [
```

Comment:
`exports` is initially assigned to `module.exports`. This is why reassigning `exports` doesn't export anything, because `exports` is now whatever you assigned it to, and not `module.exports`.

Anyways, I think it makes more sense to use `module.exports` here, because all we want to export is the array.